README.md

```diff
@@ -71,6 +72,29 @@ Configure Delphes on lxplus.cern.ch
   make
 ```
 
+Install Delphes from conda-forge
+================================
+
+Delphes is also available as a pre-built binary package from [conda-forge](https://anaconda.org/conda-forge/delphes).
+
+Command to install the package in a conda environment:
+
+```
+  conda install --channel conda-forge delphes
```

Comment:
```suggestion
conda install --channel conda-forge delphes
```

I'm not sure why these indents are here but I would strongly suggest they be removed as when rendered in Markdown they make it so that the commands are no longer usefully copy-pasteable into a terminal by clicking on the copy button on the right-hand-side.